src/commands/deploy/deploy.ts

```diff
@@ -81,6 +81,15 @@ async function deploy(actionContext: IActionContext, arg1: vscode.Uri | string |
         return await getOrCreateFunctionApp(context)
     });
 
+    if (node.contextValue.includes('container')) {
+        const learnMoreLink: string = 'https://aka.ms/deployContainerApps'
+        await context.ui.showWarningMessage(localize('containerFunctionAppError', 'Deploy is not supported for containerized function apps within the Azure Functions extension. Please use the Azure Container Apps extension to deploy your project.'), { learnMoreLink });
```

Comment:
`Please use the Azure Container Apps extension to deploy your project`

We should change this to something like. "Please read here to learn how to deploy your project" or something.

Maybe we should say Deploy is currently not supported as well since we plan to do it eventually.